config/routes.rb

```diff
@@ -16,7 +16,8 @@
   end
 
   get "dashboard" => "pages#dashboard", as: "dashboard"
-
+  post "checkout/:id" => "purchases#create", as: "checkout"
+  get "purchase_history" => "purchases#index"
```

Comment:
could we have the route here as `dashboard/purchases` please. Also move this line right after the dashboard get route.